### ISSUE TITLE
[1.8.1] Expelliarmus doesn't work

### ISSUE THREAD

**Issue by reazer18** [HarryPotterSpells/HarryPotterSpells#51]
Hello, First excuse me for my bad english, I'm french.
I noticed that the Expelliarmus spell does not work.
No error code.
The version of my server is SpigotMc-1.8.8-RO1.
Thank you in advance for your answer.
Sincerely reazer18.

**Comment by reazer18** [comment]
Up !

**Comment by jacklin213** [comment]
Hey there

Can you see if you still have the problem if you just run a server with only HarryPotterSpells plugin

Thanks

**Comment by reazer18** [comment]
Hello,

Excuser le retard, même sans plugins sauf Harry Potter Spell expelliarmus ne fonctionne pas, mais est-ce que le Gamerulle KeepInventory peut affecter ?

Amicalement reazer18

**Comment by jacklin213** [comment]
Oh are you in survival game mode when you are using HarryPotterSpells, sometimes it doesn't work in creative and adventure mode

**Comment by reazer18** [comment]
Yes i'm in survival mode, I tested with all games modes, I test the plugin in my public server, then all players test the spell Expelliarmus in survival mode.

**Comment by jacklin213** [comment]
I just tested it and it seems to work. When you use it does the particle effect go off? Do other spells work?

**Comment by reazer18** [comment]
The other spells It Works. And yes the particles triggered.
